### PR TITLE
Atc::Aws::S3Uploader now sets content_type when uploading files

### PR DESCRIPTION
**Pull request by elohanlon**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -45,6 +45,7 @@
 /config/atc.yml
 /config/aws.yml
 /config/gcp.yml
+/config/gcp-credentials.json
```
Comment (elohanlon) [inline .gitignore:48]: This ignore is in preparation for an upcoming branch.  Wanted to get it in as early as possible though.

**Review by fcd1 — APPROVED** [review]
Looks good, but I'm not as familiar with this code as Ben is.

File: .gitignore (hunk 1 above)
Comment (elohanlon) [reply .gitignore]: I ended up removing this .gitignore line in favor of storing the same info in our existing gcp.yml file (which is already ignored), so we no longer need a `gcp-credentials.json` file.

**Comment by elohanlon** [comment]
@fcd1 Thanks!

**Pull request merged**